frontend/apps/crates/components/src/audio/mixer/mixer.rs

```diff
@@ -36,8 +36,27 @@ pub struct AudioSettings {
 }
 
 impl AudioSettings {
-    pub fn reset_from_jig(&mut self, _jig: &JigData) {
-        //TODO...
+    pub fn reset_from_jig(&mut self, jig: &JigData) {
```

Comment:
This was previously unimplemented. However, I don't think that JIG-related data belongs in the audio_mixer. I'll give this a bit more thought.

Reply:
I missed earlier that when a jig is loaded for play/edit, the audio effects are set in the mixer. Before the call to set_from_jig would do nothing, but now it sets the correct sounds.